routes.yaml

```diff
@@ -0,0 +1,5 @@
+app_name: 3pa-register-admin-web
+group: web
```

Comment:
should this group be `adminweb`? 
- I'm looking at Billy's design page
- though has he done what he needs to do?

Reply:
I am not sure either Phil, will verify with Billy

Reply:
eventually this will need to be internalweb but I can leave it as web for now. I have modified the route based on feedback from Billy